docs/src/selectors.md

```diff
@@ -625,6 +644,25 @@ converts `'//html/body'` to `'xpath=//html/body'`.
 `xpath` does not pierce shadow roots
 :::
 
+## React selectors
+
+React selectors allow selecting elements by its component name. If component
+ has multiple root elements (see [React Fragments](https://reactjs.org/docs/fragments.html)),
+ `react` selector will return all of them.
```

Comment:
I don't quite understand "all of them". When I click on react=MultiRoot, will it click all of them? Let's be clear here. Also, I'd move this to a note, since it's an edge case.

Reply:
dropped this until we release `_nth` engine